mw_transform_jq.go

```diff
@@ -53,15 +53,9 @@ func (t *TransformJQMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Re
 }
 
 func transformJQBody(r *http.Request, t *TransformJQSpec, contextVars bool) error {
-	// Read the body:
-	defer r.Body.Close()
```

Comment:
did you delete this on purpose?

Reply:
If you are asking, I guess it's because I shouldn't have. I thought the json decoder was doing the close()

Reply:
The json decoder can't (or rather shouldn't) do the closing as it receives an `io.Reader`, not an `io.ReadCloser`.